cockatrice/src/messagelogwidget.cpp

```diff
@@ -508,139 +506,357 @@ void MessageLogWidget::logRevealCards(Player *player, CardZone *zone, int cardId
     else
         cardStr = cardLink(cardName);
 
-    QString str;
     if (cardId == -1) {
-        if (otherPlayer) {
-                    str = tr("%1 reveals %2 to %3.");
-            appendHtmlServerMessage(str.arg(sanitizeHtml(player->getName())).arg(zone->getTranslatedName(true, CaseRevealZone)).arg(sanitizeHtml(otherPlayer->getName())));
-        } else {
-            appendHtmlServerMessage((
-              tr("%1 reveals %2.")
-            ).arg(sanitizeHtml(player->getName()))
-             .arg(zone->getTranslatedName(true, CaseRevealZone)));
-        }
+        if (otherPlayer)
+            appendHtmlServerMessage(QString(tr("%1 reveals %2 to %3."))
```

Comment:
TR will return a QString itself, no need to wrap it (https://doc.qt.io/qt-5/qobject.html#tr)